### PR TITLE
Endpoint de invitar empleado

### PR DESCRIPTION
**Pull request by CamiloRamirezR**
## Summary by CodeRabbit

- **New Features**
	- Introduced a new employee invitation functionality via the `/api/v1/employees/invite` endpoint.
	- Added validation for employee invitations, including checks for admin privileges and employee status.

- **Bug Fixes**
	- Enhanced error handling for various invitation scenarios, ensuring accurate feedback for users.

- **Tests**
	- Expanded test coverage for the employee invitation process, including various edge cases and permission checks.

### CODE REVIEW

File: tests/blueprints/test_employee.py [hunk 1]
```diff
@@ -284,3 +293,120 @@ def test_invalid_page_number(self) -> None:
...
+        return Employee(
+            id=cast(str, self.faker.uuid4()),
+            client_id=client_id,
+            name=self.faker.name(),
+            email=self.faker.email(),
+            password=self.faker.password(),
+            role=self.faker.random_element(list(Role)),
+            invitation_status=InvitationStatus.UNINVITED,
+            invitation_date=datetime.now(UTC).replace(microsecond=0),
+        )
```
[the remark this answers is not in the document]
Comment (CamiloRamirezR) [reply tests/blueprints/test_employee.py]: I pushed a fix in commit 5e5c1bf, please review it.

File: tests/blueprints/test_employee.py [hunk 2]
```diff
@@ -284,3 +293,120 @@ def test_invalid_page_number(self) -> None:
         self.assertEqual(resp.status_code, 400)
         resp_data = json.loads(resp.get_data())
         self.assertEqual(resp_data, {'code': 400, 'message': 'Invalid page_number. Page number must be 1 or greater.'})
+
+    """
+    Invite Employee tests
+    """
```
[the remark this answers is not in the document]
Comment (CamiloRamirezR) [reply tests/blueprints/test_employee.py]: I pushed a fix in commit 5e5c1bf, please review it.

**Review by tpambor — APPROVED** [review]
